### PR TITLE
Fix/card link refactor

### PR DESCRIPTION
**Pull request by nicklee**

### CODE REVIEW

**Comment by nicklee** [comment]
This fixes the issue of having two repeated links for every card when a link is provided.

**Comment by nicklee** [comment]
Ticket: https://projects.torchbox.com/projects/rca-website-rebuild/tickets/665

**Comment by kevinhowbrook** [comment]
This will need rebasing off master when picked up

**Review by chris-lawton — APPROVED** [review]

File: rca/project_styleguide/templates/patterns/molecules/card/card--project.html [hunk 1]
```diff
@@ -4,10 +4,10 @@
         {% image item.image fill-80x58 as image_small %}
         {% image item.image fill-392x284 as image_large %}
     {% endif %}
-    <div class="card__container">
-        {% if item.link %}<a href="{{ item.link }}" class="card__image-container" aria-label="{{ item.title }}">{% endif %}
+    <{% if item.link %}a href="{{ item.link }}"{% else%}div{% endif %} class="card__container">
```
Comment (chris-lawton) [inline rca/project_styleguide/templates/patterns/molecules/card/card--project.html:7]: Nitpick: spacing on the `{% else%}`

**Pull request merged**